packages/common/src/components/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { PropsWithChildren } from 'react';
+
+import { Spinner, SpinnerProps } from '@patternfly/react-core';
+
+type LoadingSpinnerProps = PropsWithChildren &
+  SpinnerProps & {
+    isLoading: boolean;
+  };
+
+export const LoadingSpinner: React.FC<LoadingSpinnerProps> = ({
```

Comment:
Thanks!

Please namely import FC

Reply:
Please add default export to the component and remove the export form LoadingSpinner , and please delete the index.ts; we will eventually remove them all, as they can cause circular dependency. - it's all in our doc btw